doc/source/developer.rst

```diff
@@ -45,6 +45,7 @@ So that a ``pandas.DataFrame`` can be faithfully reconstructed, we store a
 .. code-block:: text
 
    {'index_columns': ['__index_level_0__', '__index_level_1__', ...],
+    'column_index_names': [<column index level name 0>, <column index level name 1>, ...],
```

Comment:
should also store: ``columns_index_dtypes`` to faithfully restore the dtypes as well

Reply:
this one to?